snap/local/build.sh

```diff
@@ -39,4 +39,4 @@ docker run \
   -w "/certbot" \
   -e "PIP_EXTRA_INDEX_URL=http://localhost:8080" \
   "adferrand/snapcraft:${DOCKER_ARCH}-stable" \
-  snapcraft
+  bash -c "snapcraft clean && snapcraft"
```

Comment:
nit: Why not?
```suggestion
snapcraft clean
snapcraft
```
or
```suggestion
snapcraft clean && snapcraft
```

Reply:
For both forms would actually call `snapcraft` in the host, not in the docker. `docker run` expect only one executable and its parameters.

Reply:
Oh! I misread this as just calling `snapcraft` after the previous command. You're right.